app/policies/rent_policy.rb

```diff
@@ -1,19 +1,11 @@
 class RentPolicy < ApplicationPolicy
   def create?
-    user.present? && user == record.user
```

Comment:
Why do you delete this line? This one was perfect 😢

Reply:
Put back :D